docs/README.md

```diff
@@ -13,17 +13,18 @@ If you plan to build a webapp to run on Cozy, you'll probably want to use a simp
 
 ## CSS Styleguide
 
-Check the [styleguide](https://cozy.github.io/cozy-ui/styleguide) to see all the variables, mixins, classes, utilities and how to use them.
+Check the [styleguide](https://cozy.github.io/cozy-ui/styleguide) to see all the variables, mixins, classes, utilities and how to use them with CSS classes only
 
 ## React components
 
-Check our [UI components](https://cozy.github.io/cozy-ui/react/) to see how to use ready made components.
+Check our [UI components](https://cozy.github.io/cozy-ui/react/) to see how to use ready made React components.
```

Comment:
s/Check our [UI components](https://cozy.github.io/cozy-ui/react/) to see how to use ready made React components./Check out [UI components](https://cozy.github.io/cozy-ui/react/) to see how to use ready made (p)React components./